src/PrestaShopBundle/EventListener/Admin/Context/ShopContextListener.php

```diff
@@ -234,7 +276,7 @@ private function verifyRouteAttribute(Request $request): ?ShopConstraint
             } else {
                 return null;
             }
-        } catch (NoConfigurationException) {
+        } catch (NoConfigurationException|ReflectionException) {
             return null;
         }
```

Comment:
Yes we do because not all controllers respect this format, especially this was failing on the web debug routes when multishop was setup Because some controllers use a service name that is simply invoked I guess